etc/workers.conf

```diff
@@ -120,4 +120,3 @@ reference-list:
     taglib-gm: 1            # loads tags using a worker (performance thing)
     talklib-gm: 1           # loads comments using a worker (performance thing)
     t-memlimit: 1           # for running certain tests
-    userpic-resize-gm: 1    # resizes userpics
```

Comment:
I am not sure this file is actually attached to anything anymore... we certainly don't use it, I don't even know what parses this. So, I think nothing happens either way.